src/todo.js

```diff
@@ -0,0 +1,75 @@
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+function updateLocalStorage() {
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+function addTodo(decription) {
+  const newTodo = { decription, completed: false, index: todos.length + 1 };
+  todos.push(newTodo);
+  updateLocalStorage();
+  return newTodo;
+}
+
+function removeTodo(index) {
+  index -= 1;
+  todos = todos
+    .filter((todo, ind) => index !== ind)
+    .map((todo, index) => ({
+      ...todo,
+      index: index + 1,
+    }));
+
+  updateLocalStorage();
+}
```

Comment:
- [ ] There is a problem happening while removing todos, for example, if you have 4 todos in your list and you remove the todo that has id=2 all todos that have id= 2, 3, and 4 are deleted from the localstorage and when you reload the page they disappear from the dom too and only one todo is left which has id = 1. and If you removed the todo of id=1 all todos got deleted. Please fix that.

![chrome-capture-2023-1-22](https://user-images.githubusercontent.com/68796232/220658906-af170132-b0ea-445b-9d78-c7afba5ab96a.gif)